examples/example_resources/lib/gen/assets.gen.dart

```diff
@@ -153,7 +160,7 @@ class SvgGenImage {
     WidgetBuilder? placeholderBuilder,
     String? semanticsLabel,
     bool excludeFromSemantics = false,
-    SvgTheme? theme,
+    SvgTheme theme = const SvgTheme(),
```

Comment:
The default `SvgTheme` has been removed since 063458a4c5c1928715fe26409cb92eebee2e5e82. Could you confirm that you are generating files using the latest code?